.github/workflows/ci.yml

```diff
@@ -114,37 +76,24 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Setup PHP version
+        id: setup-php
         uses: shivammathur/setup-php@v2
         with:
           php-version: ${{ env.PHP_VERSION }}
           coverage: none
 
-      - name: Pull composer vendor cache
-        uses: actions/cache/restore@v3
-        with:
-          path: vendor
-          fail-on-cache-miss: true
-          key: composer-build-dev-${{ hashFiles('**/composer.lock') }}
-
-      - name: Load php-cs-fixer cache
-        id: php-cs-fixer-cache
-        uses: actions/cache/restore@v3
+      - name: Cache php-cs-fixer
+        uses: actions/cache@v3
         with:
           path: .php-cs-fixer.cache
-          key: php-cs-fixer-${{ github.ref_name }}
-          restore-keys: |
-            php-cs-fixer-
+          key: ${{ steps.setup-php.outputs.php-version }}-php-cs-fixer-${{ hashFiles('.php-cs-fixer.php') }}
+          restore-keys: ${{ steps.setup-php.outputs.php-version }}-php-cs-fixer-
 
-      - name: Check code style
-        run: composer cs
+      - name: Install dependencies
+        uses: ramsey/composer-install@v2
 
-      - name: Push php-cs-fixer branch cache
-        id: push-php-cs-fixer-cache-branch
-        if: always()
-        uses: actions/cache/save@v3
-        with:
-          path: .php-cs-fixer.cache
-          key: php-cs-fixer-${{ github.ref_name }}
+      - name: Check code style
+        run: composer run cs
```

Comment:
`composer cs` looks like a Composer command, but in reality it is a Composer script